runtime/src/bank.rs

```diff
@@ -6881,7 +6881,7 @@ impl Loader for Bank {
             .ok()
     }
 
-    fn get_account_shared_data(&self, pubkey: &Pubkey) -> Option<AccountSharedData> {
+    fn load_account(&self, pubkey: &Pubkey) -> Option<AccountSharedData> {
```

Comment:
How about `get_account()` instead? It's up to the implementation of the trait if they load it or fetch it from a cache.